integr8ly/tutorial-web-app
# INTLY-145 Add PF4 masthead component

## Motivation
Fixes INTLY-145, updating the web app's masthead to use PF4 components.

## What
Replaces PF3 components that comprise the masthead to PF4 components.

## Why
Part of the initiative to update the webapp from PF3 components to PF4 components.

## How
Added new PF4 PageHeader components within the integreatly-web-app masthead component to replace PF3 masthead components.

## Verification Steps
1. Go to any page with a masthead in the web app.
2. Verify that the PF4 masthead appears, with title image and user drop-down.
3. Verify that the actions occur when you click the title (go to first page of webapp) and the drop-down (logout current user).

## Checklist:

## Progress

## Additional Notes
Landing page with PF4 masthead:
![pf4-masthead](https://user-images.githubusercontent.com/39063664/52067830-6ba00400-2549-11e9-9e54-ea85c5238f30.png)
